udata/auth/mails.py

```diff
@@ -27,3 +27,13 @@ def send_mail(self, template, subject, recipient, sender, body, html, user, **kw
         msg.html = html
 
         sendmail.delay(msg)
+
+    def normalize(self, email):
```

Comment:
Could you add a small comment on this custom normalize/validate for future udata users? :)